usrp_drivers/n200/n200_driver.cpp

```diff
@@ -634,29 +658,41 @@ void control(zmq::context_t* driver_c) {
 
 
 
+
+/**
+ * @brief      UHD wrapped main function to start threads.
+ *
+ * @return     EXIT_SUCCESS
+ *
+ * Creates a new multi-USRP object using parameters from config file. Starts control, receive,
+ * and transmit threads to operate on the multi-USRP object.
+ */
 int UHD_SAFE_MAIN(int argc, char *argv[]) {
   GOOGLE_PROTOBUF_VERIFY_VERSION;
+  uhd::set_thread_priority_safe();
 
-  auto driver_options = std::make_shared<DriverOptions>();
+  DriverOptions driver_options;
 
-  std::cout << driver_options->get_device_args() << std::endl;
-  std::cout << driver_options->get_tx_rate() << std::endl;
-  std::cout << driver_options->get_pps() << std::endl;
-  std::cout << driver_options->get_ref() << std::endl;
-  std::cout << driver_options->get_tx_subdev() << std::endl;
+  std::cout << driver_options.get_device_args() << std::endl;
```

Comment:
Do we want these as debug statements? Or at least a TODO to fix where the output is going? probably don't want the output to go to stdout

Reply:
I just changed these to debug statements.